comixed-adaptors/src/test/java/org/comixedproject/adaptors/comicbooks/ComicFileAdaptorTest.java

```diff
@@ -292,14 +293,27 @@ public void testIsComicFileForCB7() {
 
   @Test
   public void testFindAvailableFilename() {
-    final String result = adaptor.findAvailableFilename("src/test/resources/notfound", 0, "cbz");
+    final String result =
+        adaptor.findAvailableFilename(
+            TEST_ORIGINAL_FILENAME, "src/test/resources/notfound", 0, "cbz");
 
     assertEquals("src/test/resources/notfound.cbz", result);
   }
 
+  @Test
+  public void testFindAvailableWithOriginalFilename() {
+    final String result =
+        adaptor.findAvailableFilename(
+            TEST_ORIGINAL_FILENAME, "src/test/resources/example", 0, "cbz");
+
+    assertEquals("src/test/resources/example-2.cbz", result);
+  }
+
```

Comment:
this test find available filename with original filename ?

Reply:
I ask because this part of the code is not covered. 
if (file.exists() && StringUtils.pathEquals(originalFilename, candidate)) {
      log.debug("Using original filename: {}", originalFilename);
      return originalFilename;
}

I was wondering if it was an error in the test or a test oversight :)

Reply:
I've fixed the tests to make sure it covers this scenario and pushed the update.